src/app/Http/Controllers/RegisterController.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace App\Http\Controllers;
+
+
+use App\Models\User;
+use App\Http\Requests\RegisterRequest;
+
+class RegisterController extends Controller
+{
+    public function view(){
+        return view('auth/register');
+    }
+
+    public function logout() {
```

Comment:
add return types

Reply:
I removed the logout function for now Sir, Thank you!